### ISSUE TITLE
/mooculus/whatIsALimit/exercises/exerciseList

### ISSUE THREAD

**Issue by sarahmgebroff** [XimeraProject/server#304]
I keep correctly answering the questions on problem 7 in "What is a Limit?" and it will not save my answers and give me credit.

**Comment by kisonecat** [comment]
Have you tried clicking "Erase" at the top of the page?  What browser are you on?

I'm available today in Math Tower 658 until 1:30pm if you have time to show me what is happening in person.

**Comment by kisonecat** [comment]
What browser are you using?

**Comment by sarahmgebroff** [comment]
I’m using safari. I’ve tried it on both my MacBook and IPad.

**Comment by kisonecat** [comment]
Safari's support for WebSockets is a little iffy -- can you try Firefox or Chrome on a MacBook and let me know how it goes?

(I wrote some new code to fix Safari, but I am worried about deploying it until I have a chance to test it further.)

**Comment by sarahmgebroff** [comment]
Now I can no longer access Ximera in general. The page won't load on either
server or device.

**Comment by kisonecat** [comment]
I just restarted the server -- can you try again?

**Comment by sarahmgebroff** [comment]
It worked! What should I do if this happens again. But even on chrome,
Ximera will not save my answer.

**Comment by kisonecat** [comment]
You mean the page is loading in Chrome, but the answer isn't even saved even after clicking Erase?  (Could you come by Bart Snapp's office at 4:15pm?  He is in Math Building 220.)

**Comment by sarahmgebroff** [comment]
Yes that is correct. But I’m sorry I cannot because I’m working at my job
tonight.
